dash-pipeline/Makefile

```diff
@@ -610,7 +610,6 @@ docker-pull-saichallenger-client-bldr:
 	docker pull $(DOCKER_SAI_CHALLENGER_CLIENT_BLDR_IMG)
 
 docker-saichallenger-client:
-	cd $(SAI_CHALLENGER_PATH) && ./build.sh -i client
```

Comment:
The `dash-saichallenger-client` image uses `dash-saichallenger-client-bldr` as a base image but not `sc-client`.